metrics.md

```diff
@@ -221,25 +223,18 @@ In the metrics below, we adopt the following nomenclature:
 
 ### Brier Score (BS)
 {: .anchor }
-The BS measures the accuracy of forecast probability for one or more events:
+The BS measures the accuracy of forecast probability for one or more events ([Brier50](#ref-brier50)). For events with binary outcomes, BS is defined as:
 
 $$ \text{BS} = \frac{1}{n} \sum_{i=1}^n (f_i - o_i)^2  $$
 
-Smaller values of BS indicate better agreement between forecasts and observations.
-
+Smaller values of BS indicate better agreement between forecasts and observations. Note that while BS can be generalized to events with more than two outcomes, the Solar Forecast Arbiter only includes built-in support for the (more commonly used) binary events definition.
```

Comment:
Suggest adding a reference Wilks Section 7.4.2 at the end of this paragraph.

Reply:
Good idea. I'll add it now.

Reply:
I changed to: `... support for the (more commonly used) binary events definition. For more info, see Section 7.4.2. of [Wilks11](#ref-wilks11).`

Thoughts?